src/main/core/support/configuration.rs

```diff
@@ -724,15 +724,13 @@ const ONE_GBIT_SWITCH_GRAPH: &str = r#"graph [
   node [
     id 0
     ip_address "0.0.0.0"
-    country_code "XX"
     bandwidth_up "1 Gbit"
     bandwidth_down "1 Gbit"
   ]
   edge [
     source 0
     target 0
-    latency 1.0
-    jitter 0.0
```

Comment:
I think we currently documented jitter as unused, but do we validate the latency value that gets configured? If we validate a specific format, then it would be better to make sure the format is in the correct (ie, follows the same format as latency). But if we don't validate it, then I think we can just ignore the format for now (since we documented jitter as unused).

Reply:
Right now if 'jitter' is provided, shadow checks that it's numeric, so maybe we should change that to a string? And we do verify the format of 'latency', but it requires a unit. If we want to validate the 'jitter' field, what unit do we want it to use?

Reply:
I'm wondering if it's better to allow it to be added to the graph but not validate it since I don't think it's used anywhere internally in Shadow right now. That way they can put whatever they want there and we ignore it for now, and maybe someday we can start using it.

Reply:
Sounds good to me. Added in #1415.